### PR TITLE
[SPARK-25503][CORE][WEBUI]Total task message in stage page is ambiguous

### PR DESCRIPTION
**Pull request by shahidki31**
## What changes were proposed in this pull request?
Test steps : 
   1) bin/spark-shell --conf spark.ui.retainedTasks=10
   2) val rdd = sc.parallelize(1 to 1000, 1000)
  3)  rdd.count

Stage page tab in the UI will display 10 tasks, but display message is wrong. It should reverse.
 
**Before fix :** 
webui_1

**After fix**
spark_web_ui2

## How was this patch tested?

Manually tested

### CODE REVIEW

**Comment by shahidki31** [comment]
cc @vanzin

File: core/src/main/scala/org/apache/spark/ui/jobs/StagePage.scala [hunk 1]
```diff
@@ -132,7 +132,7 @@ private[ui] class StagePage(parent: StagesTab, store: AppStatusStore) extends We
     val totalTasksNumStr = if (totalTasks == storedTasks) {
       s"$totalTasks"
     } else {
-      s"$storedTasks, showing ${totalTasks}"
+      s"$totalTasks, showing $storedTasks"
```
Comment (dongjoon-hyun) [inline core/src/main/scala/org/apache/spark/ui/jobs/StagePage.scala]: Could you update the title to `[SPARK-25503][CORE][WEBUI] ...`?

**Comment by dongjoon-hyun** [comment]
Retest this please.

**Review by srowen — APPROVED** [review]

File: core/src/main/scala/org/apache/spark/ui/jobs/StagePage.scala (hunk 1 above)
Comment (shahidki31) [reply core/src/main/scala/org/apache/spark/ui/jobs/StagePage.scala]: Done. Thanks

**Comment by dongjoon-hyun** [comment]
ok to test.

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM.

I also built and tested manually, but it seems that we need a Jenkins test.

**Comment by shahidki31** [comment]
Hi @dongjoon-hyun , seems Jenkins is down. could you please ask for test again?

**Review by jiangxb1987 — APPROVED** [review]

**Comment by dongjoon-hyun** [comment]
@shahidki31 . Someone seems to take care of it. This will be merged today after Jenkins passes.

**Comment by shahidki31** [comment]
seems Jenkins is down again. @SparkQA Test this please.

**Comment by dongjoon-hyun** [comment]
Retest this please.

**Comment by vanzin** [comment]
retest this please

**Comment by dongjoon-hyun** [comment]
Merged to master/branch-2.4/branch-2.3.

**Pull request closed without merging**